packages/workspace/src/browser/workspace-frontend-contribution.ts

```diff
@@ -38,12 +44,19 @@ export class WorkspaceFrontendContribution implements CommandContribution, MenuC
             isEnabled: () => true,
             execute: () => this.showFileDialog()
         });
+        commands.registerCommand(WorkspaceCommands.CLOSE, {
+            isEnabled: () => this.workspaceService.opened,
+            execute: () => this.showWorkspaceDialog()
```

Comment:
`() => this.closeWorkspace()` also would be better, otherwise it is not clear what `showWorkspaceDialog` does

Reply:
done